File: 1281-subtract-the-product-and-sum-of-digits-of-an-integer/1281-subtract-the-product-and-sum-of-digits-of-an-integer.cs
public class Solution {
    public int SubtractProductAndSum(int n) {
        int product=1;
        int sum=0;
        string str=n.ToString();
        for(int i=0;i<str.Length;i++){
            int digit=str[i]-'0';
            product*=digit;
            sum+=digit;
        }
        return product-sum;
    }
}

File: 1281-subtract-the-product-and-sum-of-digits-of-an-integer/1281-subtract-the-product-and-sum-of-digits-of-an-integer.ts
function subtractProductAndSum(n: number): number {
     let product=1;
    let sum=0;
    let str=n.toString();
    for(let i=0;i<str.length;i++){
        let digit=Number(str[i])
        product*=digit;
        sum+=digit;
    }
    return product-sum;
};